$NetBSD: patch-ad,v 1.1 2006/08/03 00:39:24 rillig Exp $

This is needed so that the preprocessor macros can be queried in
mn-md5.h.

--- src/mn-md5.c.orig	2006-08-01 11:13:36.000000000 +0200
+++ src/mn-md5.c	2006-08-01 11:13:47.255953537 +0200
@@ -26,6 +26,8 @@
  * edited for clarity and style only.
  */
 
+#include <config.h>
+
 #include <sys/types.h>
 #include <string.h>
 #include <glib.h>
